src/frontend/scss/objects/_course_glimpses.scss

```
@@ -202,7 +208,7 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
     @include sv-flex(1, 0, auto);
     margin: 0;
     padding: 0.75rem 0;
-    font-weight: bold;
+    font-weight: $font-weight-bold;
```

Why not directly pass the css font-weight value?
```suggestion
    font-weight: $font-weight;
```

---

Font thickness can change depending the font families, so this way with variables, a project could use another font and set their correct thickness values.